test/fixtures/test-package/esm-multiple-export.js

```diff
@@ -0,0 +1,13 @@
+import { PushActionPlugin, PullActionPlugin } from '@finos/git-proxy/plugin';
+
+// test multiple exports (ESM syntax)
+export default {
+  foo: new PushActionPlugin(async (req, action) => {
+    console.log('PushActionPlugin: ', action);
+    return action;
+  }),
+  bar: new PullActionPlugin(async (req, action) => {
+    console.log('PullActionPlugin: ', action);
+    return action;
+  }),
```

Comment:
Maybe add a dummy object here to validate the load is only picking up proper subclasses of the Push/PullActionPlugin class?
```suggestion
  }),
  baz: {
    exec: (async (req, action) => {
      console.log('not a real plugin object');
    }),
  },
```